modules/rest/src/main/java/org/apache/ignite/internal/rest/configuration/AbstractConfigurationController.java

```diff
@@ -39,7 +48,7 @@ public AbstractConfigurationController(ConfigurationPresentation<String> cfgPres
      * @return the presentation of configuration.
      */
     public String getConfiguration() {
-        return cfgPresentation.represent();
+        return maskSensitiveInformation("", cfgPresentation.represent());
```

Comment:
Could you override the method in a way it would take only one argument `String configuration` and call `maskSensitiveInformation("", configuration)` inside?